common4j/src/main/com/microsoft/identity/common/java/opentelemetry/OTelUtility.java

```diff
@@ -47,7 +47,19 @@ public static Span createSpan(@NonNull final String name) {
     }
 
     /**
-     * Creates a span from a parent Span Context (with shared basic attributes).
+     * Creates a span (with shared basic attributes).
+     **/
+    @NonNull
+    public static Span createSpan(@NonNull final String name, @NonNull final String callingPackageName) {
+        final Tracer tracer = OpenTelemetryHolder.getTracer(TAG);
+        return tracer.spanBuilder(name)
+                .setAttribute(AttributeName.calling_package_name.name(), callingPackageName)
+                .startSpan();
+    }
+
+    /**
+     * Creates a span from a parent Span Context (with shared basic attributes) and caller pkg name
+     * pre-populated on the span upon creation.
```

Comment:
nit :  the "pre-populated  on the span upon creation" part of it could be added on createSpan instead of here. since that is where you are pre-populating it

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2302/commits/8324f5fcfcd3018d06f542b194d705735f54ae0c